src/common/options.cc

```diff
@@ -492,6 +492,15 @@ std::vector<Option> get_global_options() {
     .add_service("common")
     .add_see_also("admin_socket"),
 
+    Option("daemon_debug", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
```

Comment:
I think it's okay to make this specific to FUSE. Let's call it `fuse_debug`... but now I see we already have `fuse_debug`!! So just set `fuse_debug` in the `src/common/config.cc` instead of `daemon_debug`.

Also, let's fix `fuse_debug` to have the below option settings. (description, flags, but not service/tag/see_also). Also change `fuse_debug` to an ADVANCED option from DEV.

Reply:
Yeah, using the `fuse_debug` is perfect to fix this.
Will fix it. Thanks.

Reply:
@lxbsz please also fix the "fuse_debug" option metadata as I mentioned in the above comment.

Reply:
Sorry missed this part, will fix it. Thanks.